black.py

```diff
@@ -149,8 +149,14 @@ class Feature(Enum):
 
 VERSION_TO_FEATURES: Dict[TargetVersion, Set[Feature]] = {
     TargetVersion.PY27: {Feature.ASYNC_IDENTIFIERS},
-    TargetVersion.PY33: {Feature.UNICODE_LITERALS, Feature.ASYNC_IDENTIFIERS},
-    TargetVersion.PY34: {Feature.UNICODE_LITERALS, Feature.ASYNC_IDENTIFIERS},
+    TargetVersion.PY33: {
```

Comment:
Is this a result of the changes from your PR? If so, I don't think that's an improvement.

Reply:
Hmm, I added trailing commas for some reason. Reverted this change. With no trailing commas the result is the same as it was.

Reply:
Hmm, I added trailing commas for some reason. Reverted this change. With no trailing commas the result is the same as it was.